lib/linguist/languages.yml

```diff
@@ -6786,6 +6786,20 @@ Valve Data Format:
   ace_mode: text
   tm_scope: source.keyvalues
   language_id: 544060961
+Velocity Template Language:
+  type: markup
+  color: "#507cff"
+  aliases:
+  - vtl
+  - velocity
+  extensions:
+  - ".vtl"
+  - ".velocity"
```

Comment:
Please remove this extension. I've not included any samples for it and it's nowhere near popular enough yet.

Reply:
> Please remove this extension. You've not included any samples for it and it's nowhere near popular enough yet.

098f77c7c9158095b402916a6cdb1ee3307dd2ef